src/main/scala/api.scala

```diff
@@ -127,9 +133,11 @@ case object api {
     val command: BC
   )(
     val outputRecord: OR
-  )(implicit
-    val validRecord: OR#Properties CheckForAll ValidOutputRecordFor[BC]
   )
+  // TODO fix this!
+  // (implicit
+  //   val validRecord: OR#Keys CheckForAll ValidOutputRecordFor[BC]
+  // )
```

Comment:
If I understand right, this is commented out just because in cosas-0.8 there were no `all`/`any` for predicates. Right?